source/admin_manual/debugging/index.rst

```diff
@@ -6,7 +6,7 @@ Debugging
 
 Each ``IMAP``, ``POP3`` and ``LMTP`` connection has its own unique session ID.
 This ID is logged in all the lines and passed between Dovecot services, which
-allows tracking it all the way through directors to backends and their various
+allows tracking it all the way through proxies to backends and their various
 processes.
 
 The session IDs look like ``<ggPiljkBBAAAAAAAAAAAAAAAAAAAAAAB>``
```

Comment:
add into .github/actions/spelling/patterns.txt

Reply:
Why does the spell-check pass even without it being there?..

Reply:
dunno, but it would be safer to add it there.

Reply:
ah, it's already there, at least partially...

.github/actions/spelling/patterns.txt:BBAAAAAAAAAAAAAAAAAAAAAAB

Reply:
maybe this is fine.